src/ganon/acmd/specials/special_air_side_fall.rs

```diff
@@ -1,99 +1,137 @@
 //! This function is for the dive hitbox for piledrive
-use crate::{ganon::utils::GANON_DOWN_SPECIAL_AIR_DURATION_FLAG, imports::*};
+use crate::{
+    ganon::utils::{
+        GANON_DOWN_SPECIAL_AIR_CONTINUE_FLAG, GANON_DOWN_SPECIAL_AIR_COUNTDOWN_INT,
+        GANON_DOWN_SPECIAL_AIR_MULTIPLIER_FLAG,
+    },
+    imports::*,
+};
 
+unsafe extern "C" fn use_weak_hitbox(agent: &mut L2CAgentBase) -> bool {
+    if !WorkModule::is_flag(agent.module_accessor, GANON_DOWN_SPECIAL_AIR_CONTINUE_FLAG) {
+        let countdown =
+            WorkModule::get_int(agent.module_accessor, GANON_DOWN_SPECIAL_AIR_COUNTDOWN_INT);
+        return countdown > 0 && countdown <= 10;
+    }
+    false
+}
+
+/// specialairsfall is on a 19 frame cycle. At any time in those 19 frames, the player
+/// can release the special button to transition into slowdown state of down special air,
+/// and should be reflected in the hitboxes.
 unsafe extern "C" fn ganon_specialairsfall(agent: &mut L2CAgentBase) {
```

Comment:
Is it fine that once the "19" (18) frame window is over, it maintains the strength of the hitboxes even if you let go of the special button? For example, if you hold the special button, and you let go of the button once those "19" frames are up, it will still be the strong hitbox and he will then transition into the slowdown state.

Reply:
Addressed! Once the "weak" hitbox happens, we will not trigger the damage multiplier